components/inputs/docs/input-date-time.md

```diff
@@ -112,82 +112,110 @@ To make your usage of `d2l-input-date-time` accessible, use the following proper
 
 The `input-date` and `input-time` components do not handle timezone and so require the input to be in the user's timezone, which corresponds to the timezone in the `data-timezone` attribute on the `html` element.
```

Comment:
I don't think we should mention the `data-timezone` attribute... really it's an internal thing that nobody should know about except the platform components that deal with it. Instead, you could just mention "... in the user's timezone as specified in their account settings."